cms/views.py

```diff
@@ -108,24 +108,12 @@ def filter_partners(request, partners):
                         listed_value = False
                     partners = partners.filter(Q(**{query: listed_value}))
 
-        distinct_partners = list(
-            partners.filter(query_list).order_by("-always_featured", "?")
-        )
         partners_json = json.dumps(
             serialize(
-                distinct_partners,
-                fields=[":all"],
-                exclude=[
-                    "created_on",
-                    "updated_on",
-                    "dedicated_partner_page",
-                    "id",
-                    "created_by",
-                    "updated_by",
-                ],
```

Comment:
Do these no longer need excluding somehow?

Reply:
Some fields no longer exist in the models:
- "created_on"
- "updated_on"
- "created_by"
- "updated_by"

And the others seem to be included in the resulting [json](https://partners.ubuntu.com/partners.json) anyway:
- "dedicated_partner_page"
- "id"